sda-commons-dependencies/build.gradle

```diff
@@ -14,6 +14,7 @@ ext {
   swaggerCoreV3Version = '2.1.5'
   weldVersion = '3.1.5.Final'
   openTracingUtilVersion = '0.33.0'
+  jsonUnitVersion = '2.19.0'
```

Comment:
Is there a reason why we don't use [2.21.0](https://search.maven.org/artifact/net.javacrumbs.json-unit/json-unit/2.21.0/bundle). If there is one, can we add a comment to avoid upgrade by mistake?

Reply:
Correct. We would have got a seprarate version from https://github.com/SDA-SE/sda-dropwizard-commons/pull/636. I'll update it in this PR.